layers.go

```diff
@@ -1759,6 +1763,9 @@ func (r *layerStore) tspath(id string) string {
 // layerHasIncompleteFlag returns true if layer.Flags contains an incompleteFlag set to true
 // The caller must hold r.inProcessLock for reading.
 func layerHasIncompleteFlag(layer *Layer) bool {
+	if layer.Flags == nil {
+		return false
+	}
 	// layer.Flags[…] is defined to succeed and return ok == false if Flags == nil
```

Comment:
Non-blocking nit: If the condition is added explicitly above, this comment can be dropped. (I don’t care at all which one remains, just that having both is unnecessary.)

Reply:
Dropping the comment, then.